site/en/reference/cli/contribute.md

```diff
@@ -342,6 +326,43 @@ CLI commands support the following Doxygen ALIASES and special commands:
 | @moreinfo | @moreinfo{@netdata} | Creates a referral link. |
 | @note | @note Important callout. | Creates a note callout box. |
 
+## Fixing broken lines from OpenThread `make pretty` script
+
+Occasionally, the `make pretty` script causes long lines to break incorrectly.
+This can happen with a long list of parameters or long lines of command output
+that go beyond the column width limit that the `make pretty` check imposes. 
```

Comment:
We should be clear that this is for code comments only, as well.

So maybe word it like:

"Some code comments, such as CLI parameters or command output, need to be on a single line for them to render correctly in the openthread.io reference, but `make pretty` imposes a column width limit, which breaks the rendered output for long lines."